spec/requests/reservations_spec.rb

```diff
@@ -33,11 +33,21 @@
     end
   end
 
-  describe 'POST /reservations' do
+  describe 'POST /reservations/online' do
     it 'creates new reservation and returns status 201' do
-      post('/reservations', params: { reservation: { seance_id: seance.id, ticket_desk_id: 1 } })
+      post('/reservations/online',
+           params: { seance_id: seance.id, tickets: [{ seat: 'C5', ticket_type_id: 1 }] })
       expect(response.status).to eq(201)
     end
+
+    it 'fails to create a taken seat in reservation and returns status 422' do
+      reservation = create(:reservation, seance_id: seance.id)
```

Comment:
This is a place for [contexts](https://www.betterspecs.org/#contexts) :wink:

You can reorganize it in the following way:

```
context `when ticket is already sold`
  let(:reservation) { create(:reservation, seance_id: seance.id) }
  let(:ticket) { create(:ticket, reservation_id: reservation.id)
  before { ticket }

   it 'fails to create a taken seat in reservation and returns status 422' do
     post('/reservations/online',           params: { seance_id: seance.id, tickets: [{ seat: ticket.seat, ticket_type_id: 1 }] })
  expect(response.status).to eq(422)
   end

end
```